api/napi/Cargo.toml

```diff
@@ -26,6 +26,7 @@ i-slint-core = { workspace = true, features = ["default"] }
 slint-interpreter = { workspace = true, features = ["default", "display-diagnostics"] }
 spin_on = "0.1"
 css-color-parser2 = { workspace = true }
+itertools = "0.11"
```

Comment:
That's a candidate for a workspace dependency I guess :) (not related to this PR though)

Reply:
True :) https://github.com/slint-ui/slint/pull/3667